README.md

```diff
@@ -256,6 +256,7 @@ The following options are also available in `explore()` to customize the plot:
 
 Applies to both barplots and bubble plots:
 - `space`: whether or not the axes in different panels are `fixed` or `free` (see ggplot2 docs)
+- `wrap`: whether or not the facet wrap is `veritcal` or  `horizontal` (i.e. the organization of the panes)
```

Comment:
@NickiShaw My major comment is that the `wrap = "horizontal"` setting can make plots that look a bit misleading. You are not really supposed to directly compare the normalized HMM hit levels between different HMMs due to known HMM hit biases. Normalized levels can be "off" between HMMs by a factor of 2-3x or more. I suppose the `horizontal` setting could be helpful in select situations, but you should mention in the docs here that normalized hit levels should generally not be directly compared between different HMMs (or at least, any comparisons should be taken with a grain of salt) given the potential for HMM hit bias.